cfpb/consumerfinance.gov
# TDP: Replace hardcoded grade-level icons with our standard icon loader

## Changes

- TDP: Replace hardcoded grade-level icons with our standard icon loader
- Replace standard colors with color variables.
- Minor HTML tidying.

## How to test this PR

1. Visit http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/assess/survey/ and click on each grade level and note the icons should be unchanged to those reached via https://www.consumerfinance.gov/consumer-tools/educator-tools/youth-financial-education/assess/survey/


## Screenshots

<img width="220" alt="Screen Shot 2023-03-17 at 3 33 20 PM" src="https://user-images.githubusercontent.com/704760/226003229-0ccc4faa-8e63-4ac2-86eb-9a21c483e481.png">
